tests/test_kernelbuilder.py

```diff
@@ -188,3 +188,37 @@ def test_getrelease_regex_fail(self, mock_prepare):
         with mock_popen, mock_prepare:
             with self.assertRaises(Exception):
                 self.kbuilder.getrelease()
+
+    @mock.patch("skt.kernelbuilder.KernelBuilder.adjust_config_option")
```

Comment:
Comment on the commit message: Please use imperative mood: write "Add tests ..." (or similar), not just "New tests ...". This way it's easier to reason about the commit when reverting/merging. Thanks!

Reply:
Fixed.